src/commands/__tests__/ssh.test.ts

```diff
@@ -84,9 +85,9 @@ describe("ssh", () => {
     });
 
     it("should call p0 request with reason arg", async () => {
-      void sshCommand(yargs()).parse(
-        `ssh some-instance --reason reason --provider aws`
-      );
+      void sshCommand(yargs())
+        .fail(noop)
```

Comment:
no action here but we can also use `false` for `.fail()` to get the no-op behavior.

> Providing false as a value for fn can be used to prevent yargs from exiting and printing a failure message.

Reply:
O.o